app/admin/organizations.rb

```diff
@@ -0,0 +1,34 @@
+ActiveAdmin.register Organization do
+  menu parent: 'Main App'
+
+  permit_params :name, :employees, :teams
+
+  index do
+    id_column
+    column :name
+    column :created_at
+    column :updated_at
+    actions
+  end
+
+  show do
+    attributes_table do
+      row :id
+      row :name
+      row :employees
+      row :created_at
+      row :updated_at
+    end
+  end
+
+  filter :name
+  filter :created_at
+  filter :updated_at
+
+  form do |f|
+    f.inputs do
+      f.input :name
+    end
```

Comment:
Acá se podrían poner los usuarios como input para mejorar la UX o no?

Me tinca que es más simple agregar a usuarios a una organización editandola antes que crear las relaciones `Employee` directamente:

```suggestion
    f.inputs do
      f.input :name
      f.input :users
    end
```

Igual esto requiere que en tu `permit_params` le pases como último argumento un `user_ids: []`, para que pesque.

Te parece?